<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoDesigner</class>
 <widget class="QDialog" name="DialogoDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIncrementar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Incrementar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecrementar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrementar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>120</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbUnidades">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidades</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDecenas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decenas</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBoxCantidad">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogoDesigner</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogoDesigner</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
